core/src/main/java/com/novoda/noplayer/model/TextCues.java

```diff
@@ -12,7 +12,7 @@ public TextCues(List<Cue> cues) {
         this.cues = cues;
     }
 
-    List<Cue> cues() {
+    public List<Cue> cues() {
```

Comment:
It wouldn't need to be if we collapsed some more packages but all of our models now belong in the `model` package meaning they need to be `public`. Personally, I would find it strange if we didn't expose our `NoPlayer` models.